shell/platform/darwin/ios/framework/Source/FlutterEngine.mm

```diff
@@ -393,7 +393,6 @@ - (void)setViewController:(FlutterViewController*)viewController {
 
 - (void)attachView {
   self.iosPlatformView->attachView();
-  [_textInputPlugin.get() setupIndirectScribbleInteraction:self.viewController];
```

Comment:
I don't think this line should be belong to `attachView`, I move it to `addInternalPlugins`, which is also called every time in viewDidLoad.